lib/apiservers/engine/backends/backends.go

```diff
@@ -212,8 +196,18 @@ func ProductVersion() string {
 	return productVersion
 }
 
+// VchConfig gets the VIC appliance VM's configspec
 func VchConfig() *config.VirtualContainerHostConfigSpec {
-	return vchConfig
+	var vchConfig config.VirtualContainerHostConfigSpec
```

Comment:
why do you need to reload the appliance vm configuration? It has been set during Init.
the reason I ask this is because this step will assume portlayer is running in VCH VM, then it cannot be used by vic-machine